styles/style.css

```diff
@@ -75,7 +77,8 @@ nav {
 .nav-item__logo {
     gap: 12px;
     color: black;
-
+    font-size: 27px;
```

Comment:
the font of hyperleaflet is small by intention. 
can you remove font-size and weight.